### ISSUE TITLE
pypi distribution

### ISSUE THREAD

**Issue by JulienPeloton** [astrolabsoftware/fink-client#27]
**Describe the issue**
Once the dashboard is ready (#18 ), we should distribute the client via pypi.

Action item: add a setup.py, and upload fink-client on pypi. Update the doc accordingly.

**Comment by JulienPeloton** [comment]
Not needed yet.
